### PR TITLE
create the community heroes page.

### PR DESCRIPTION
**Pull request by nisarhassan12**
This is how it looks: 

image

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1252"></a>

### CODE REVIEW

**Comment by pawlean** [comment]
2021-11-16 at 10 53 07 - Google Chrome

The community page at `/community` needs to be updated with information of the Community Heroes page, any ideas here @ChristinFrohne? I was thinking of the following mock-up (feel free to create an actual mock-up, Christin):

- Change Ambassador program to Gitpod Community Heroes
- have the 3D illustration 
- Change text to:
> Join us empowering developers to make engineering collaborative and joyful again. Are you our next community hero?
- Add button `Find out more` that links to `/community/heroes` 

Wdyt?

**Comment by pawlean** [comment]
Other than the above, the code looks good, the page looks good, thanks for bringing this to life @nisarhassan12! Excited to launch. 🚀

**Review by pawlean — COMMENTED** [review]
Other than the above, the code looks good, the page looks good. 

Thanks for bringing this to life @nisarhassan12 @ChristinFrohne! Excited to launch. 🚀

**Comment by ChristinFrohne** [comment]
>The community page at /community needs to be updated with information of the Community Heroes page, any ideas here @ChristinFrohne? 

There is already a [mockup](https://www.figma.com/file/ for it. I just updated the text with your suggestion 🙌

Bildschirmfoto 2021-11-16 um 12 19 15

**Comment by pawlean** [comment]
> > The community page at /community needs to be updated with information of the Community Heroes page, any ideas here @ChristinFrohne?
> 
> There is already a [mockup](https://www.figma.com/file/ for it. I just updated the text with your suggestion 🙌
> 
> Bildschirmfoto 2021-11-16 um 12 19 15

Already one step ahead. 😉 This is perfect, thanks! Shall we include it in this PR or open another one? I guess that's on @nisarhassan12's call but it'd be good to keep all community things in one place!

**Review by ChristinFrohne — COMMENTED** [review]
- [x] how present would you like this page to be @pawlean ? Should there be a link in the footer? I checked with auth0 and Gitkraken and they don't have it in the footer. 

Great implementation @nisarhassan12, regarding responsive design we need to become more thorough though. Some feedback:
- [ ] could you try to reduce the spacing one step between the CTA and the next section.. I feel 160px is a bit too far away. On smaller screens it already works great.
Bildschirmfoto 2021-11-16 um 12 30 30

- [x] In Figma we discussed using 18px for the text in the buttons. Here you are using 16px. Let's me sure it's decided and be consistent

- [x] Can you make "connect on Discord" a grey button. Otherwise we have too close too many primary CTAs

- [x] on mobile: use the full width for text (see also #1253 ). Should be around 343px. 
Bildschirmfoto 2021-11-16 um 12 35 09

- [x] on mobile: any reason why you use 316px width for the boxes instead of the usual 343px? The text above has a width of 343px and the call-out section as well. We need to be more consistent. The boxes should be aligned to that
Bildschirmfoto 2021-11-16 um 12 38 50

 
 
 Note: On mobile, it's usually is best to have the text first followed by an image e.g. when explaining the features on the home screen, but here it works well the other way around. Great thought!

**Comment by nisarhassan12** [comment]
@ChristinFrohne now that the community page is being updated we no longer use the signup for the ambassador program widget anywhere would it make sense to get rid of the relevant code + this sheet https://docs.google.com/spreadsheets/d/1DrgFXvDsLmu6kUoBHMfgYqBtqyT9kr4OHqcKX31FbXw/edit#gid=1742220594 as there are literally no signups apart from some test data Or do we plan to re-use this at some point in future ? Thanks

**Comment by ChristinFrohne** [comment]
>would it make sense to get rid of the relevant code + this sheet https://docs.google.com/spreadsheets/d/1DrgFXvDsLmu6kUoBHMfgYqBtqyT9kr4OHqcKX31FbXw/edit#gid=1742220594

Yes you can remove it

**Comment by nisarhassan12** [comment]
> Yes you can remove it

Thanks. I have opened this https://github.com/gitpod-io/website/issues/1260 issue will tackle that in a separate PR.

**Comment by nisarhassan12** [comment]
> could you try to reduce the spacing one step between the CTA and the next section. I feel 160px is a bit too far away. On

@ChristinFrohne Below the header element and between different sections on larger screens it is 160px as per the specs. See the headers on the following pages for instance:

- https://www.gitpod.io/gitpod-vs-github-codespaces
- https://www.gitpod.io/support
- https://www.gitpod.io/security

image

Do we want to change it globally ? for all the headers + the sections ?

**Comment by ChristinFrohne** [comment]
>Below the header element and between different sections on larger screens it is 160px as per the specs. See the headers on the following pages for instance:

It's not consistent. For other pages like on /community it's 64px, for /startups it's 72px. 
Solution: Can we consistently use 72px?

There are many other inconsistencies like button size, spacing between buttons and text, etc. Once you have time, could you look into aligning all these details? (in a separate PR)

**Comment by nisarhassan12** [comment]
> It's not consistent. For other pages like on /community it's 64px, for /startups it's 72px.
Solution: Can we consistently use 72px?

Let's discuss this in our 1:1 later today.

**Review by pawlean — COMMENTED** [review]
Can we change the title of the page to `Community Heroes`? 

2021-11-18 at 16 44 08 - Google Chrome

There is a few wording I want to change. I might jump on this PR and update them, if that's OK?

**Comment by nisarhassan12** [comment]
> Can we change the title of the page to Community Heroes?
@pawlean I have added the title. You can for sure take over this PR.

File: src/routes/community/heroes.svelte [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  import Eligibility from "../../components/eligibility.svelte";
+  import MeetTheHeros from "../../components/heroes/meet-the-heros.svelte";
+  import Explore from "../../components/explore.svelte";
+  import { exploreContents } from "../../contents/community/hereos";
+</script>
+
+<OpenGraph
+  data={{
+    title: "Community Heroes",
+    description: "", // TODO fill in the description
```
Comment (nisarhassan12) [inline src/routes/community/heroes.svelte:15]: @pawlean Not sure what a good page description can be here. Please add one. Thanks

Comment (pawlean) [reply src/routes/community/heroes.svelte:15]: Yes, will add it with my push

**Review by pawlean — COMMENTED** [review]
I edited the wording a little bit and made it more clear! Other than that, it looks ready to go to me. @ChristinFrohne WDYT?

Also related tasks that I've done to push this forwards:

- [x] Add Front integration - tested that responses will auto-assign to me in Front, if I'm OOO they will 'live' in the Community Heroes inbox
- [x] Ambassador guide has been written
- [ ] Form has been updated (thanks to Christin's feedback)

...And that's everything! LMK if I missed anything, but feeling pretty good to launch this early next week.

**Comment by ChristinFrohne** [comment]
Just checked the page on my phone and some things are still not aligned. Have you tested it on mobile @nisarhassan12 ?

- [ ] the cards don't take the full width
IMG_7927

- [ ] the icons are blurry

IMG_7929
- [ ] the content of the card should be centered to be consistent with the card style above it
IMG_7928

Once that's fixed, it's approved by me :)

**Comment by nisarhassan12** [comment]
>  the cards don't take the full width

Thanks @ChristinFrohne They do when the device width is 400px and smaller. In Figma, we have 375px as the base width for the mobile layout see https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=1%3A683

 If I make them take up full width at devices with a width less than or equal to 600px(the following screenshot is at device-width 550px). Maybe it's just me but In my opinion, it does not look good i.e now the cards are too wide and the length of the line is a bit long and they no longer have their square shape that looks good for the cards. We have some Phones that have a width higher than 600px. I was wondering what to do in that case. 

image

Below is how they look when the device width is 400px or less in this case they look totally fine as the square shape is preserved and the length of the line is not too long. 

image

Mobile devices come in different sizes and dimensions these days we should aim to have the UI look fine across all the different device widths. 

Should I make them span full width at max-width 768px or 600px ?


> - [ ]  the icons are blurry

Not sure why is that they look fine on my phone I'm running Chrome on Android. 

image

**Comment by trumbitta** [comment]
> there are literally no signups apart from some test data

Changed that situation a couple seconds ago 👀

**Comment by nisarhassan12** [comment]
@ChristinFrohne Please re-review this again and look at my comments above. Would be great if we can merge this as soon as possible to avoid merge conflicts with work that I'm currently doing in another PR.

**Comment by ChristinFrohne** [comment]
@nisarhassan12 I already said that once you went through my last feedback, I'm approving the PR. Regarding the responsive design would be great if you can look at other pages and see how they handle it. Further optimization on  the responsive design can also be done in #1253

**Comment by 0Grit** [comment]
image

There is a typo in the google form.

**Comment by pawlean** [comment]
> image
> 
> There is a typo in the google form.

Good shout. Thanks @0Grit! Let me edit that. :)

**Pull request merged**